### ISSUE TITLE
Loads external resources

### ISSUE THREAD

**Issue by legoktm** [APerson241/vote-history#2]
```
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/locale/en-gb.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
```

Please don't use an external CDN - it violates the privacy of people just clicking on links without really realizing it. All the libraries you need should be available from https://tools.wmflabs.org/cdnjs/.

**Comment by APerson241** [comment]
Huh, I didn't even know that existed or that we had to use the internal CDN. Converting it right now.

**Comment by APerson241** [comment]
Fixed in b294327a

**Comment by legoktm** [comment]
Thanks!
